modules/configuring-ragas-remote-provider-for-production.adoc

```diff
@@ -0,0 +1,398 @@
++
+[source,terminal]
+----
+$ oc login <openshift_cluster_url> -u <username> -p <password>
+----
+
+. Navigate to your data science project:
++
+[source,terminal]
+----
+$ oc project <project_name>
+----
+
+. Create a secret for storing S3 credentials:
++
+[source,terminal]
+----
+$ oc create secret generic "<ragas_s3_credentials>" \
+  --from-literal=AWS_ACCESS_KEY_ID=<your_access_key> \
+  --from-literal=AWS_SECRET_ACCESS_KEY=<your_secret_key> \
+  --from-literal=AWS_DEFAULT_REGION=<your_region>
+----
++
+[IMPORTANT]
+====
+Replace the placeholder values with your actual S3 credentials. These AWS credentials are required in two locations:
+
+* In the Llama Stack server pod (as environment variables) - to access S3 when creating pipeline runs.
+* In the Kubeflow Pipeline pods (via the secret) - to store evaluation results to S3 during pipeline execution.
+
+The LlamaStackDistribution configuration loads these credentials from the 
+`"<ragas_s3_credentials>"` secret and makes them available to both locations.
+====
+
+. Create a secret for the Kubeflow Pipelines API token:
+.. Get your token by running the following command:
++
+[source,terminal]
+----
+$ export KUBEFLOW_PIPELINES_TOKEN=$(oc whoami -t)
+----
+.. Create the secret by running the following command:
++
+[source,terminal]
+----
+$ oc create secret generic kubeflow-pipelines-token \
+  --from-literal=KUBEFLOW_PIPELINES_TOKEN="$KUBEFLOW_PIPELINES_TOKEN"
+----
++
+[IMPORTANT]
+====
+The Llama Stack distribution service account does not have privileges to create pipeline runs. This secret provides the necessary authentication token for creating and managing pipeline runs.
+====
+
+. Verify that the Kubeflow Pipelines endpoint is accessible:
++
+[source,terminal]
+----
+$ curl -k -H "Authorization: Bearer $KUBEFLOW_PIPELINES_TOKEN" \
+ https://$KFP_ENDPOINT/apis/v1beta1/healthz
+----
++
+You should see a successful health check response confirming that the endpoint is accessible with your token.
+
+. Create a secret for storing your inference model information:
++
+[source,terminal]
+----
+$ export INFERENCE_MODEL="llama-3-2-3b"
+$ export VLLM_URL="https://llama-32-3b-instruct-predictor:8443/v1"
+$ export VLLM_TLS_VERIFY="false"  # Use "true" in production
+$ export VLLM_API_TOKEN="<token_identifier>"
+
+$ oc create secret generic llama-stack-inference-model-secret \
+  --from-literal INFERENCE_MODEL="$INFERENCE_MODEL" \
+  --from-literal VLLM_URL="$VLLM_URL" \
+  --from-literal VLLM_TLS_VERIFY="$VLLM_TLS_VERIFY" \
+  --from-literal VLLM_API_TOKEN="$VLLM_API_TOKEN"
+----
+
+. Get the Kubeflow Pipelines endpoint URL. This URL is used in a later step for creating a ConfigMap for the Ragas remote provider configuration:
++
+[source,terminal]
+----
+$ export KFP_ENDPOINT=$(oc get route -n <project_name> -l app=ds-pipeline-dspa -o jsonpath='{.items[0].spec.host}')
+
+$ echo "Kubeflow Pipelines endpoint: https://$KFP_ENDPOINT"
+----
```

Comment:
And is this OK? cc @adolfo-ab @dmaniloff

Reply:
Thanks, I updated the step, which you can see [here](https://github.com/opendatahub-io/opendatahub-documentation/pull/1028/files#diff-463c41326cbed380810f18d06d6085ce089a69ef1f75673cf97633fccb0459deR136-R143).